src/docker-in-docker/install.sh

```diff
@@ -10,7 +10,7 @@
 
 DOCKER_VERSION="${VERSION:-"latest"}" # The Docker/Moby Engine + CLI should match in version
 USE_MOBY="${MOBY:-"true"}"
-MOBY_BUILDX_VERSION="${MOBYBUILDXVERSION}"
+MOBY_BUILDX_VERSION="${MOBYBUILDXVERSION:-"latest"}"
```

Comment:
> *I'm using the latest file (as of now) https://github.com/devcontainers/features/blob/eea8ec3a3702057ce3c0c3e54bbb205ed580867b/src/docker-outside-of-docker/install.sh

From https://github.com/devcontainers/features/issues/865

Curious, @lucasfcnunes why are you using the script directly instead of using Dev Container Features? We don't provide guarantee or support that using scripts would always work.

Can you try updating your dev container to add Feature? See https://github.com/devcontainers/features/blob/main/src/docker-outside-of-docker/README.md#example-usage

Reply:
It's faster for the people using the devcontainer. We pre-build the image before distributing it.

Reply:
You can still prebuild the image with a `devcontainer.json` (topped with Features). Have you tried that? 

All the images hosted from https://github.com/devcontainers/images uses the same pattern, see https://github.com/devcontainers/images/blob/main/src/universal/.devcontainer/devcontainer.json

Reply:
> You can still prebuild the image with a devcontainer.json (topped with Features). Have you tried that?

I have. In the past.

Summarizing I do 4+ things that don't go so well with traditional devcontainer builds:

```Dockerfile
# 1. bleeding edge custom syntax (COPY --link, RUN --mount, ...)
# syntax=docker/dockerfile:1

# 2. heavy caching
RUN rm -f /etc/apt/apt.conf.d/docker-clean; echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache

RUN \
  --mount=type=cache,target=/var/cache/apt/,sharing=locked \
  --mount=type=cache,target=/var/lib/apt/,sharing=locked \
  {DooD, kubectl-helm-minikube}.sh # after `perl -p -i -e` out cache cleanse ('s|rm -rf /var/lib/apt/lists/\*|# <remove apt-cache>|g')

# 3. lots of `COPY --link` in multistage build (~9 FROM ... AS ...).
COPY --link ... ...

```

```
# 4. build all of it with just 01 `docker buildx bake` run (no `docker build` nor `docker compose build`)
```

---

I'm already setting the `MOBYBUILDXVERSION` env var in my build args to `latest` - so all good by my side.
This commit would close my initial issue and follow the tradition of all vars having default values.

Reply:
> follow the tradition of all vars having default values.

Sooner or later we will be cleaning up this stale tradition as it's not required for the dev container Features to work. Also, we don't recommend or support the use of script files.

A heads up, we can allow this PR contribution for now, but this could break at any point of time.